PanoptesNetClient/PanoptesNetClient/ApiClient.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Newtonsoft.Json.Linq;
+using System;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading.Tasks;
+
+namespace PanoptesNetClient
+{
+    public class Project
```

Comment:
Not sure if this is normal for C#, but it seems odd that you are defining multiple classes in this file. Even if `Project` is small, you are probably going to end up having a few other data models in this client and they probably shouldn't all go in here.

Reply:
Ah yes, I need to remove this good catch. This class isn't currently being used and it was initially added as I was trying to figure out the best way to begin serializing/deserializing the json objects.